main/src/ca/uwaterloo/flix/language/ast/TokenKind.scala

```diff
@@ -255,6 +279,10 @@ object TokenKind {
 
   case object LiteralChar extends TokenKind
 
+  case object LiteralDebugStringL extends TokenKind
+
+  case object LiteralDebugStringR extends TokenKind
+
```

Comment:
What's this?

Reply:
Those are tokens for the start and end of a interpolated debug string
```
"%{ myValue }"
 ^^         ^
```

Reply:
Ok